XSOAR CI CD playbook to support GitLab (#15855)
demisto/content: Packs/ContentManagement/ReleaseNotes/1_0_9.md

#### Incident Fields

> - **Configuration File Source**: added option to choose Gitlab as source

Reviewer: Add rn to the PB as well.

Reviewer: ```suggestion
- **Configuration File Source** 
Added the *Gitlab* option to the single select options.
```

#### Incident Fields
- **Configuration File Source** 
Added the *Gitlab* option to the single select options.

#### Playbooks
##### Configuration Setup

> - Getting file from gitlab if gitlab is selected

Reviewer 2: ```suggestion
- Added the option of getting a file from GitLab if **Gitlab** is selected.
```
